internal/pkg/agent/cmd/run.go

```diff
@@ -188,7 +192,7 @@ func run(override cfgOverrider, modifiers ...component.PlatformModifier) error {
 		l.Info("APM instrumentation disabled")
 	}
 
-	coord, err := application.New(l, logLvl, agentInfo, rex, tracer, configuration.IsFleetServerBootstrap(cfg.Fleet), modifiers...)
+	coord, err := application.New(l, baseLogger, logLvl, agentInfo, rex, tracer, configuration.IsFleetServerBootstrap(cfg.Fleet), modifiers...)
```

Comment:
Do we need to pass into to loggers? Can we just use the `l` with the `component.binary` added, across all of the Elastic Agent.

Because here https://github.com/elastic/elastic-agent/blob/main/pkg/component/runtime/command.go#L507 we do not depend on a base logger, so fields like this are added to the main logger it will not affect a component runtime.